content/en/docs/languages/rust/libraries.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Using instrumentation libraries
+linkTitle: Libraries
+weight: 40
+description: How to instrument libraries an app depends on
+---
+
+{{% docs/languages/libraries-intro Rust %}}
+
+## Use Instrumentation Libraries
+
+Each instrumentation library is a [crate](https://crates.io/).
+
+For example,
+[the instrumentation library for Actix Web](https://crates.io/crates/actix-web-opentelemetry)
+will automatically create [spans](/docs/concepts/signals/traces/#spans) and [metrics](/docs/concepts/signals/metrics) based on
+the inbound HTTP requests.
+
+You can find a list of available instrumentation libraries in the
+[registry](/ecosystem/registry/?language=js&component=instrumentation).
```

Comment:
It might be good to add https://github.com/tokio-rs/tracing-opentelemetry << Since in the rust community you have many crates which are already instrumented with the `tracing` crate from `tokio-rs` this allows for wiring those into OpenTelemetry.

Reply:
Good call out, thank you, I can add a few words around that too!